src/Illuminate/Support/Str.php

```diff
@@ -546,17 +545,7 @@ public static function isJson($value)
             return false;
         }
 
-        if (function_exists('json_validate')) {
-            return json_validate($value, 512);
-        }
-
-        try {
-            json_decode($value, true, 512, JSON_THROW_ON_ERROR);
-        } catch (JsonException) {
-            return false;
-        }
-
-        return true;
+        return json_validate($value, 512);
```

Comment:
Can't we remove the `512` here? That's the default and we don't include it in the changes in `ValidatesAttributes.php` in this PR.